### ISSUE TITLE
Function implementation: check if Pump is faulty

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#66]
## User story

1. As a facility manager
2. I want to SDs to implement a function for faulty pumps
3. So that I can identify if pumps are faulty

## Acceptance criteria

- [ ] The function checks all pumps of a facility
- [ ] It sets a pump to faulty if the flow is over time below a constant threshold value (e.g. 5 l/s)
- [ ] If flow drops towards 0, temperature rises, and flow goes back to normal, the pump is not faulty but suspicious 

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release

**Comment by sn0wcat** [comment]
Acceptance Criteria is wrong @lamara11 

- [ ] The pump is suspicious if the flow drops towards 0, temperature rises, and flow goes back to normal
- [ ] The pump is faulty if the flow is over time below a constant treshold value (e.g. 5 l/s)

**Comment by lamara11** [comment]
@sn0wcat  I edited the acceptance criteria!
